networking/aws_load_balancer_operator/installing-albo-sts-cluster.adoc

```diff
@@ -1,12 +1,12 @@
 :_content-type: ASSEMBLY
 [id="albo-sts-cluster"]
-= Installing AWS Load Balancer Operator on Secure Token Service cluster
+= Installing AWS Load Balancer Operator on Security Token Service cluster
```

Comment:
Should this be "a Security Token Service cluster"?

Reply:
Probably :)